### PR TITLE
Tweaks to the UI text

### PR DESCRIPTION
**Pull request by lubinalubina**
Various tweaks to the UI text.

### CODE REVIEW

File: src/hwdevice.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ def prepare_transfer_tx(self, caller, bip32_path,  utxos_to_spend, dest_address,
             pubkey_hash = bin_hash160(curr_pubkey)
             pubkey_hash_from_script = extract_pkh_from_locking_script(prev_transaction.outputs[utxo_tx_index].script)
             if pubkey_hash != pubkey_hash_from_script:
-                text = "Error: different public key hashes for the BIP32 path and the UTXO"
+                text = "Error: The hashes for the public key for the BIP32 path, and the UTXO do not match."
                 text += "locking script. Your signed transaction will not be validated by the network.\n"
```
Comment (random-zebra) [inline src/hwdevice.py]: Need to change both lines.
```
text = "Error: The hashes for the public key for the BIP32 path, and the UTXO locking script do not match."
text += "Your signed transaction will not be validated by the network.\n"
```

File: src/qt/gui_tabMain.py [hunk 2]
```diff
@@ -96,36 +96,36 @@ def insert_mn_list(self, name, ip, port, row=None):
         ##--- Details button
         self.btn_details[name] = QToolButton()
         self.btn_details[name].setIcon(self.details_icon)
-        self.btn_details[name].setToolTip('Check STATUS details of masternode "%s"' % name)
+        self.btn_details[name].setToolTip('Check status of "%s"' % name)
```
Comment (random-zebra) [inline src/qt/gui_tabMain.py]: this is the magnifying glass. it is for *details*.
The next button is just for *status*

File: src/qt/gui_tabRewards.py [hunk 3]
```diff
@@ -36,7 +36,7 @@ def initRewardsForm(self):
         self.mnSelect = QComboBox()
         self.mnSelect.setToolTip("Select Masternode") 
         hBox.addWidget(self.mnSelect)
-        label = QLabel("Address Balance")
+        label = QLabel("Total Balance")
```
Comment (random-zebra) [inline src/qt/gui_tabRewards.py]: `Total` could be misleading here.
One could think it's the total balance of a masternode (collater + rewards).

For example, creating 2 masternodes with the same collateral address is perfectly legal (provided they have different UTXOs).

**Review by random-zebra — CHANGES_REQUESTED** [review]
just few minor points

**Comment by lubinalubina** [comment]
Updated all comments, and pushed changes

**Review by random-zebra — APPROVED** [review]

**Pull request merged**